diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/DiktatJavaExecTaskBase.kt

```diff
@@ -68,10 +68,10 @@ open class DiktatJavaExecTaskBase @Inject constructor(
                 }
                 .files
                 .forEach {
-                    add("\"${it.path}\"")
+                    add(it.path)
```

Comment:
Why is this necessary?

Reply:
Seems that ktlint doesn't remove quotes from path patterns, they should be removed by a shell when invoking it from CLI, but in JavaExec they are not needed.